baseplate/experiments/__init__.py

```diff
@@ -90,6 +97,30 @@ def _get_experiment(self, name):
             self._experiment_cache[name] = experiment
         return self._experiment_cache[name]
 
+    def get_all_experiment_names(self):
+        """Returns a list of all valid experiment names from the
```

Comment:
nit: [pep-257](https://github.com/reddit/styleguide/blob/master/python/pep-0257.rst) says these should be as a command like "Return a list" rather than "Returns a list". The summary (first line) should fit on one line if possible as well.